astropy/units/quantity_helper.py

```diff
@@ -449,6 +449,8 @@ def helper_degree_minute_second_to_radian(f, unit1, unit2, unit3):
 
 try:
     import scipy.special as sps
+    # RTD Fix: See https://github.com/astropy/astropy/issues/7159
+    from scipy.special import loggamma  # noqa
```

Comment:
This will remove units support for all other `special` methods though probably most of them are available in scipy < 0.18. You could probably handle this specific import in a special way and append to the list below if it is available?

Reply:
maybe introduce a `SCIPY_LT_018` variable?
Anyway, please make a note that the workaround is needed for scipy <0.18 support, so we will have a chance to clean-up once we move beyond that version.

Reply:
Good points. I'll make the requested changes. Thank you both!